ts/screens/wallet/AddPaymentMethodScreen.tsx

```diff
@@ -55,41 +63,56 @@ const getpaymentMethods = (
   {
     name: I18n.t("wallet.methods.card.name"),
     description: I18n.t("wallet.methods.card.description"),
+    icon: CreditCard,
     onPress: props.navigateToAddCreditCard,
     status: "implemented",
     section: "credit_card"
   },
   {
-    name: I18n.t("wallet.methods.postepay.name"),
-    description: I18n.t("wallet.methods.postepay.description"),
-    onPress: props.navigateToAddCreditCard,
-    status: "implemented",
-    section: "credit_card"
+    name: I18n.t("wallet.methods.paypal.name"),
+    description: I18n.t("wallet.methods.paypal.description"),
+    icon: PaypalLogo,
+    // TODO replace with effective dispatch
+    onPress: constNull,
+    status: payPalEnabled ? "implemented" : "notImplemented",
+    section: "digital_payments"
+  },
+  {
+    name: I18n.t("wallet.methods.bancomatPay.name"),
+    description: I18n.t("wallet.methods.bancomatPay.description"),
+    icon: BpayLogo,
+    status: !onlyPaymentMethodCanPay ? "implemented" : "notImplemented",
+    onPress: props.startBPayOnboarding,
+    section: "digital_payments"
+  },
+  {
+    name: I18n.t("wallet.methods.satispay.name"),
+    description: I18n.t("wallet.methods.satispay.description"),
+    icon: SatispayLogo,
+    onPress: props.startSatispayOnboarding,
+    status: !onlyPaymentMethodCanPay ? "implemented" : "notImplemented",
```

Comment:
```suggestion
    status: "notImplemented",
```
In order to keep Satispay disabled.